scripts/robotstuff.coffee

```diff
@@ -0,0 +1,142 @@
+# coppied from https://www.npmjs.com/package/hubot-robotstuff since it was broken as a package
```

Comment:
added here but removed from external-scripts?
are external-scripts not in the repo and scripts that are in the repo don't have to be listed on external-scripts?

Reply:
yep